### PR TITLE
app-misc/geoclue: version bump to 2.4.7

### PR DESCRIPTION
**Pull request by zozom**
Package-Manager: Portage-2.3.5, Repoman-2.3.1

### CODE REVIEW

File: app-misc/geoclue/geoclue-2.4.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
```
Comment (EvaSDK) [inline app-misc/geoclue/geoclue-2.4.7.ebuild]: Please bump to EAPI=6, in accordance with https://wiki.gentoo.org/wiki/Project:GNOME/Gnome_Team_Ebuild_Policies#EAPI

**Comment by zozom** [comment]
Tried to bump to EAPI="6"

**Comment by EvaSDK** [comment]
Looks good.

File: app-misc/geoclue/geoclue-2.4.7.ebuild [hunk 2]
```diff
@@ -13,7 +12,7 @@ SRC_URI="https://www.freedesktop.org/software/${PN}/releases/${MY_PV}/${P}.tar.x
 
 LICENSE="LGPL-2"
 SLOT="2.0"
-KEYWORDS="~alpha ~amd64 ~arm ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd"
+KEYWORDS="~alpha amd64 ~arm ~ia64 ~mips ~ppc ~ppc64 ~sparc x86 ~amd64-fbsd"
```
Comment (monsieurp) [inline app-misc/geoclue/geoclue-2.4.7.ebuild]: No no no. Please mark back `amd64` and `x86` unstable.

File: app-misc/geoclue/geoclue-2.4.7.ebuild [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+DEPEND="${RDEPEND}
+	dev-util/gdbus-codegen
+	>=dev-util/gtk-doc-am-1
+	>=dev-util/intltool-0.40
+	sys-devel/gettext
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	epatch "${FILESDIR}"/${PN}-2.4.1-fix-GLIBC-features.patch
```
Comment (monsieurp) [inline app-misc/geoclue/geoclue-2.4.7.ebuild]: If this ebuild goes EAPI version 6, patches must be defined in the `PATCHES` array. Also `default` must be called in the `src_prepare()` phase.

File: app-misc/geoclue/geoclue-2.4.7.ebuild (hunk 2 above)
Comment (zozom) [inline app-misc/geoclue/geoclue-2.4.7.ebuild]: Oops sorry. My mistake. The keywords have been corrected.

**Comment by monsieurp** [comment]
Superseded by #4830.

**Pull request closed without merging**